code/game/objects/items/miscellaneous.dm

```diff
@@ -78,6 +243,10 @@
 			hero_item_list[initial(A.name)] = A
 	return hero_item_list
 
+/obj/item/choice_beacon/hero/spawn_option(obj/choice,mob/living/M)
+	new choice(get_turf(M))
+	to_chat(M, "<span class='hear'>You hear something crackle from the beacon for a moment before a voice speaks. \"Please stand by for a message from Sophronia Broadcasting. Message as follows: <b>Please enjoy your Sophronia Broadcasting's 'History Comes Alive branded' Costume Set, exactly as shown in the hit show!</b> Message ends.\"</span>")
+
```

Comment:
Please document all the changes you make in the pr, it's mostly so I can say gattcha if you hide a grenade in a pie somewhere or something.